domains/transfer-agreements/TransferAgreements.API/API/Models/Connection.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace API.Models;
+
+public class Connection
+{
+    public Guid Id { get; set; }
+    public Guid CompanyId { get; set; }
```

Comment:
Is the name CompanyId the best name for this property? If I am right, then this is to be populated with the SubjectId in the JWT-token - which means that it is the Id assigned by the Auth-service. So I am fearing that AuthService may call this, e.g. OrganizationId. Then we have 3 different terms for the same thing.

Reply:
CompanyId is then the subject id of the one who invited

Reply:
We discussed it - Auth is using company as well

Reply:
ok